src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -366,48 +540,39 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
         height: 44px;
       }
 
-      paper-icon-button {
-        opacity: var(--dark-primary-opacity);
+      .playPauseButton {
+        width: 56px !important;
+        height: 56px !important;
       }
 
-      paper-icon-button[disabled] {
-        opacity: var(--dark-disabled-opacity);
+      .top-info {
+        display: flex;
+        justify-content: space-between;
       }
 
-      .playPauseButton {
-        width: 56px !important;
-        height: 56px !important;
-        background-color: var(--primary-color);
-        color: white;
-        border-radius: 50%;
-        padding: 8px;
-        transition: background-color 0.5s;
+      .icon-name {
+        display: flex;
+        height: fit-content;
+        align-items: center;
       }
 
-      .playPauseButton[disabled] {
-        background-color: rgba(0, 0, 0, var(--dark-disabled-opacity));
+      .icon-name :first-child {
```

Comment:
This should only be added to `ha-icon`.
```suggestion
      .icon-name ha-icon {
```